app/routes/main.js

```diff
@@ -18,22 +20,31 @@ module.exports = function(passport) {
         res.render('index');
     });
 
-    router.get('/login', passport.authenticate('github'), function() {
+    router.get('/login', passport.authenticate('github', { session: true }), function() {
         // The request will be redirected to GitHub for authentication, so this function will not be called.
     });
 
-    router.get('/login/callback', passport.authenticate('github', { failureRedirect: '/' }), function(req, res) {
+    router.get('/login/callback', passport.authenticate('github', { failureRedirect: '/', session: true }), function(req, res) {
         res.redirect('/profile');
     });
 
     router.get('/profile', isLoggedIn, function(req, res) {
-        Log.find({ 'userId': req.user.id }, function (err, logs) {  // TODO: ajax this in instead of loading server-side
-            res.render('profile', {
-                user: req.user,
-                logs: logs
+        User.findById(req.user._id).populate('account').exec(function(err, user) {
```

Comment:
This is just telling Mongo to execute the query. What is taking place here is called population, and basically populates an entity with it's referenced account. [See here for more info](http://mongoosejs.com/docs/populate.html)